src/pages/api/users-boringqueue-withdrawals.ts

```
@@ -0,0 +1,49 @@
+import { NextApiRequest, NextApiResponse } from "next"
+
+const baseUrl =
+  process.env.NEXT_PUBLIC_BASE_URL || "http://localhost:3000"
+
+const usersBoringQueueWithdrawals = async (
+  req: NextApiRequest,
+  res: NextApiResponse
+) => {
+  try {
+    let { vaultAddress, userAddress, chain } = req.query
```

ESLint

[error] 'vaultAddress' is never reassigned. Use 'const' instead.

(prefer-const)